docs/source/contributing.rst

```diff
@@ -0,0 +1,70 @@
+Undebt_: Contributing
+==========================
+
+.. _Undebt: index.html
+.. default-role:: code
+
+Getting Started
+---------------
+
+Undebt's development is taking place on Github, so please go ahead and `fork`_ the repository if you want to begin contributing.
+
+.. _`fork`: https://github.com/Yelp/undebt#fork-destination-box
+
+You'll then want to get a local copy of the code base:
+
+.. code-block:: bash
+
+    git clone git@github.com:<your-username>/undebt.git
+
+Getting Setup
+-------------
+
+It is highly recommended that you create a `virtual environment`_ before installing the project dependencies.
+
+.. _`virtual environment`: http://docs.python-guide.org/en/latest/dev/virtualenvs/
+
+Both can be achieved by running:
+
+.. code-block:: bash
+
+    make dev
+
+Running the Tests
+-----------------
+
+Undebt uses `tox`_ for testing.
+
+.. _`tox`: https://tox.readthedocs.io/en/latest/
+
+You can run the entire test suite:
+
+.. code-block:: bash
+
+    make test
+
+Or, run an individual environment:
+
+.. code-block:: bash
+
+    tox -e py35
```

Comment:
Can you add a note that this will (probably) only work if they are inside of the virtualenv?